### ISSUE TITLE
Biothings explorer requirement pegged to an old release?

### ISSUE THREAD

**Issue by dkoslicki** [RTXteam/RTX#1303]
Looking at the [requirements](https://github.com/RTXteam/RTX/blob/master/requirements.txt#L25) it appears that we might be pegged to a specific release of BTE. Given that BTE has has more [recent releases](https://github.com/biothings/biothings_explorer/releases/tag/v1.0.3), we should make sure that the version pegging with `@` isn't to an older version.

**Comment by dkoslicki** [comment]
And actually, seems even worse: pegged to a specific [commit](https://github.com/biothings/biothings_explorer/commit/b7be98589ea5e4dfee5a80b9cada88fdac128c81)

**Comment by amykglen** [comment]
yep, it's true -  it's pegged to a specific commit. this brings up something I've been thinking about: now that there's nice general-purpose code in Expand for querying TRAPI-compliant KPs, it actually would be less work for us if we could use BTE's API (using the BTE python package means we have to do some extra parsing of the QG to get it in a different input format, and other translation steps like that that can be a little messy). 

is there any reason why we shouldn't use their API? it seems as speedy as using the python package in the little testing I've done of it. and it would basically mean no extra work for us. :)

**Comment by dkoslicki** [comment]
I don't see any reason why using the API would be detrimental, especially given the performance comparisons you bring up. Less work and more sustainability seems good to me!

**Comment by amykglen** [comment]
awesome! this is done in `master` and is running smoothly for me locally. will confirm on production when `master` is next rolled out.
